app/models/early_career_payments/eligibility_answers_presenter.rb

```diff
@@ -134,5 +135,36 @@ def itt_academic_year
         "itt-year"
       ]
     end
+
+    private
+
+    def text_for_subject_question
+      qualification_symbol = eligibility.qualification.to_sym
+
+      if subject_symbols.many?
+        I18n.t("early_career_payments.questions.eligible_itt_subject", qualification: qualification_to_substring(qualification_symbol))
+      else
+        subject_symbol = subject_symbols.first
+        I18n.t("early_career_payments.questions.eligible_itt_subject_one_option", qualification: qualification_to_substring(qualification_symbol), subject: subject_symbol)
+      end
+    end
+
+    def subject_symbols
+      cc = current_claim
+      JourneySubjectEligibilityChecker.new(claim_year: cc.policy_year, itt_year: cc.eligibility.itt_academic_year).selectable_subject_symbols(cc)
+    end
```

Comment:
I guess you could keep that assignemnt, but if you want to make the line shorter why not
```ruby
def subject_symbols
  JourneySubjectEligibilityChecker.new(
    claim_year: current_claim.policy_year,
    itt_year: current_claim.eligibility.itt_academic_year
  ).selectable_subject_symbols(current_claim)
end
```